For Advanced/030.md

```diff
@@ -0,0 +1,37 @@
+# Automate.Net
+
+You learned how to find webpage elements and perform actions on them with the `selenium.` commands, so it’s time to see some of these functions in combat.
+
+Run this script:
+
+```G1ANT
+selenium.open chrome url google.com
+selenium.type g1ant search q by name
+delay 2
+selenium.presskey enter search btnK by name
+selenium.gettext search //*[@id="rso"]/div[1]/div/div[1]/div/div/div[1]/a[1]/div/cite by xpath result ♥url
+selenium.newtab ♥url
+delay 2
+selenium.activatetab google by title
+selenium.setattribute name value value ‴robotic process automation‴ search q by name
+selenium.presskey enter search q by name
+selenium.click search selenium.click search //*[@id="rso"]/div[3]/div/div[1]/div/div/div[1]/a/h3 by xpath
+```
+
+Let’s translate this code into the natural language:
+
+1. Open Chrome and go to Google.com.
+2. Enter “g1ant” into the element named “q” (which is the search phrase input box).
+3. Wait 2 seconds.
```

Comment:
Explain in this point why the robot needs to wait. Explain to me why 2s, not 1s nor 10s.
In fact, you can't wait for a fixed amount of time. You mast wait for the page to be fully loaded. Also, I feel it is not necessary here to wait at all, because previous operation
- is performed on already loaded page,
- is not asynchronous, so there's no need to wait.

Reply:
this wait time is just to allow the user to see the effects